### ISSUE TITLE
[Feature] Seed target in WebUI

### ISSUE THREAD

**Issue by lnm83** [hadouken/hadouken#263]
I would like to be able to set a seed target through the WebUI.

I have tried following the documentation to update the JSON file to manually maintain the seed ratio/time, but it has failed to restart every time. This should be much easier through the webui.

One alternative would be to have the boilerplate included in the template to prevent issues.

**Comment by ghost** [comment]
I agree, I've also tried to configure the JSON file and failed miserably every time. An option on the web UI would make everything a lot easier.

**Comment by vktr** [comment]
That functionality is a bit broken, sorry. I know @Robo210 did a fix a while ago but it is only in the develop branch as of now.

**Comment by lnm83** [comment]
Could it be put into master?

Alternatively, it's there an msi of the dev branch?

On 2 Nov 2016 8:34 AM, "Viktor Elofsson" notifications@github.com wrote:

That functionality is a bit broken, sorry. I know @Robo210
https://github.com/Robo210 did a fix a while ago but it is only in the
develop branch as of now.

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub
https://github.com/hadouken/hadouken/issues/263#issuecomment-257705354,
or mute the thread

**Comment by vktr** [comment]
Hi! The AppVeyor build don't seem to keep artifacts after some time, so the latest dev build has none, sorry. I'll see if I can trigger a new build.
